### ISSUE TITLE
A11y_.NET Core_ASP.NETCore_AngularFetchData_ScreenReader:  NVDA is not announcing as column headers when focus lands on the column header.

### ISSUE THREAD

**Issue by Truptik12** [dotnet/aspnetcore#42505]
[42505.zip](https://github.com/dotnet/aspnetcore/files/9019573/42505.zip)

### Actual:
NVDA is not announcing as column headers when focus lands on the column header

**This issue is observed throughout the application for all the Fetch Data pages.**
**_This issue is also observed with Narrator and Chromium Edge_**

### Expected:
NVDA should announce as column headers when focus lands on the column header

“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#A11yMAS;#A11yTCS;#.NET Core[aspnetcore]-Win32-June2022;#.NET Core;#A11ySev2;#WCAG1.3.1;#DesktopWeb;#Win11;#Benchmark;#Element:ColumnHeader;#NVDA;#Narrator;

### Environment Details:
.NET Core
ASP.NET Core
Operating System: Windows 11 Enterprise 21H2
Chrome: Version 102.0.5005.115 (Official Build) (64-bit)

### Steps to Reproduce:

1. Open command prompt
2. Create a folder using command "mkdir foldername" e.g. "mkdir Angular"
3. Move to the created directory using command "cd foldername"
4. Run the following command "dotnet new angular -au Individual" and hit enter.
5. Run the following command "dotnet run"
6. Copy the First URL link from Command prompt and copy it
7. Paste it in URL section of Browser and hit Search
8. Browser page will open with above template loaded
9. Hit Enter on fetch data link .
10. Turn on the screen reader and observe the anouncements,

### User Impact:
Screen reader users will not understand if they focus is on column header or not.

**Comment by SteveSandersonMS** [comment]
I checked, and this table appears to be already as semantically correct as it can be:

![image](https://user-images.githubusercontent.com/1101362/182670844-c7d0275b-3ad5-4e8c-92a8-7e892a45618b.png)

It already uses `<th>` to identify column headers (and `<thead>`, though that's not consequential for accessibility).  The resulting screen reader behavior in NVDA is exactly the same as on other "accessible table" examples I can find on the web. The screen reader knows where the column headers are, and it's up to the screen reader to choose how to express that.

So I'm not sure what this issue is suggesting we should change. If you can give an example of how you think a table should be expressed to produce the correct screen reader behaviors, please let me know.

I'm closing this as it appears to have been reported in error, but if I'm wrong, please reopen with further details!

**Comment by mkArtakMSFT** [comment]
#Closed; as there is no issue here and if the accessibility tooling is unable to read the column headers (which are correctly generated) then it's a problem with the accessibility tooling instead.
